### ISSUE TITLE
Inverse search on mac does not work with no servername created

### ISSUE THREAD

**Issue by ThSGM** [lervag/vimtex#2813]
### Description

This issue has been resolved but I'm logging it here.

I am trying to get reverse searching working with vimtex ([u/lervag](https://www.reddit.com/u/lervag)) on macOS with Skim. Here is the [guide](https://dr563105.github.io/blog/skim-vimtex-setup/) I followed.

However, it wasn't working. Typing in: `:echo v:servername` returned blank. I managed however to get it working by running `nvim --listen tmp/nvimsocket filename.tex`.

Now the echo commands returns the name of that folder, and also the reverse searching now works. I thought that neovim should already be creating a server by default on launch. Do you know why it's not and I have to create one manually?

### Do you use a latexmkrc file?

No

### VimtexInfo

```
> nvim -V1 -v
version
nvim v0.8.2
build type: release
luajit 2.1.0-beta3
compiled by brew@monterey

features: +acl +iconv +tui
see ":help feature-compile"

   system vimrc file: "$vim/sysinit.vim"
  fall-back for $vim: "/usr/local/cellar/neovim/0.8.2/share/nvim"
```

```yaml
System info:
  OS: macOS 12.6.2 (21G320)
  Vim version: NVIM v0.8.2
  Has clientserver: true
  Servername: undefined (vim started without --servername)

VimTeX project: tmp
  base: tmp.tex
  root: /Users/tri
  tex: /Users/tri/tmp.tex
  main parser: fallback current file
  document class: 
  compiler: latexmk
    engine: -pdf
    options:
      -verbose
      -file-line-error
      -f
      -synctex=1
      -interaction=nonstopmode
    callback: 1
    continuous: 1
    executable: latexmk
  viewer: Skim
  qf method: LaTeX logfile
```

**Comment by ThSGM** [comment]
Hi, 

Just chiming in to say that I've resolved the issue but I don't know why it wasn't working in the first place. 

This thread helped me: https://github.com/neovim/neovim/issues/20865

In particular, the command:

    :edit $NVIM_LOG_FILE

indicated that it was unable to start a server due to not having the folder $XDG_RUNTIME_DIR which was my home folder with .xdg. 

I created this folder with mkdir and now the servers start up and the reverse searching works. I don't know why I didn't have this folder in the first place.
